index.html

```diff
@@ -250,6 +131,51 @@ <h2 class="contactme-title">Contact me</h2>
         </section>
     </div>
 
+     <!-- Popup section -->
+     <div class="popupcont">
+        <div class="popupcard">
+            <div class="pophead">
+                <img class="popclose" id="closeModal" src="./Images/Works/close.png">
+                <h2 class="poptitle">Tonic</h2>
+                <div class="popdetail">
+                    <p class="company">CANOPY</p>
+                    <img src="./Images/Works/Counter.png" alt="">
+                    <p class="jobdescription">Back End Dev</p>
+                    <img src="./Images/Works/Counter.png" alt="">
+                    <p class="year">2015</p>
+                </div>
+            </div>
+            <div class="popimg">
+                <img class="popimge mob" src="./Images/Works/SnapshootPortfolio1.png">
+                <img class="popimge desk" src="./Images/Works/Snapshoot-Portfolio-Desktop.png">
+            </div>
+            <div class="sect">
+            <div class="popdescription">
+                <p class="popdescrip">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```

Comment:
## Mobile Popup
### Your Mobile Popup
![figma-mobile (copy)](https://user-images.githubusercontent.com/93039359/189059543-3c87fafa-1bad-41cd-b879-b37aadb6a9f0.png)

### Figma Mobile Popup
![figma-mobile-popup](https://user-images.githubusercontent.com/93039359/189056423-e1570b41-d0ef-45d9-acfa-4f1be3923e08.png)

## Desktop Popup
### Your Desktop Popup
![popup-student](https://user-images.githubusercontent.com/93039359/189056449-a9001582-1b8f-4f27-9dbd-4f98bb537dec.png)

### Figma Desktop Popup
![figma-popup (another copy)](https://user-images.githubusercontent.com/93039359/189059607-3446ec87-ca5a-41ab-a5b0-6c31a0ddc453.png)

- [ ] Kindly, add the grey overlay in your design because you are missing it. Please add it to follow the Figma template :blush:
- [ ] Your popup is not matching with the Figma popup. Please make your mobile and desktop popup look similar to the Figma template :blush:
- [ ] Your Popup is not showing up on clicking any other button than the first project. Please fix it :blush:
- [ ] Your Popup is not scrollable. Please make it scrollable so we would be able to see all the contents on the popup window :+1: